### ISSUE TITLE
OpenMax baseline experimental configuration in Few-shot setting

### ISSUE THREAD

**Issue by MinkiJ** [BoLiu-SVCL/meta-open#2]
Hi, thank you for sharing your experimental code.

In the paper, the OpenMax baseline uses the negative of distances like ProtoNet.

But, the lower $\omega$ value in OpenMax indicates the query sample is more likely to be an unknown sample.
Therefore, I think the OpenMax method assumes all elements in the activation vector to be non-negative.
However the logits of ProtoNet are non-positive, which is inconsistent with the assumption.

Also, since there is a single sample for each class in 1-shot cases, it is nearly impossible to fit the Weibull model using supports as stated in the OpenMax paper (e.g. The OpenMax code uses 20 samples for each class to fit the Weibull model).

Due to those problems, I could not reproduce your result on the baselines.

Could you give more detailed information to reproduce your baselines (Openmax & Counterfactual)?

Thank you!
